sysproxy2go.sh

```diff
@@ -0,0 +1,36 @@
+###############################################################################
+#
+# This script regenerates the source files that embed the sysproxy-cmd executable.
+#
+###############################################################################
+
+function die() {
+  echo $*
+  exit 1
+}
+
+if [ -z "$BNS_CERT" ] || [ -z "$BNS_CERT_PASS" ]
+then
+  die "$0: Please set BNS_CERT and BNS_CERT_PASS to the bns_cert.p12 signing key and the password for that key"
+fi
+
+BINPATH=../sysproxy-cmd/binaries
+
+# Check for a recent version of osslsigncode that can handle 32-bit Windows
```

Comment:
We need to check for a recent osslsigncode that can handle 32 bit windows binaries.